inference-engine/src/transformations/src/transformations/rt_info/fused_names_attribute.cpp

```diff
@@ -51,24 +51,23 @@ std::vector<std::string> ngraph::getFusedNamesVector(const std::shared_ptr<ngrap
     const auto &rtInfo = node->get_rt_info();
     using FusedNamesWrapper = VariantWrapper<FusedNames>;
 
-    if (!rtInfo.count(FusedNamesWrapper::get_type_info_static().name)) return {};
+    if (!rtInfo.count(FusedNamesWrapper::get_type_info_static())) return {};
 
-    const auto &attr = rtInfo.at(FusedNamesWrapper::get_type_info_static().name);
+    const auto &attr = rtInfo.at(FusedNamesWrapper::get_type_info_static());
     FusedNames fusedNames = ov::as_type_ptr<FusedNamesWrapper>(attr)->get();
     return fusedNames.getVectorNames();
 }
 
 template class ov::VariantImpl<FusedNames>;
 
-BWDCMP_RTTI_DEFINITION(VariantWrapper<FusedNames>);
```

Comment:
Is it really not needed?

Reply:
Yes, we don't use type_info in all rt info attributes now.